endtoend_tests/forseti/conftest.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Forseti end-to-end test configuration"""
+
+import pytest
+import time
+from endtoend_tests.helpers.forseti_cli import ForsetiCli
+
+
+@pytest.fixture
+def forseti_cli():
+    return ForsetiCli()
+
+
+@pytest.fixture
+def forseti_inventory_readonly(forseti_cli):
```

Comment:
Added new fixtures for the tests to share. For example, the CV policy tests will create a single inventory/scan and assert violations with that one scan. Previous tests were creating a different inventory/scan for each test. This will help performance.